shared/actions/profile/proofs.js

```diff
@@ -44,31 +45,28 @@ const recheckProof = (state, action) =>
     Tracker2Gen.createShowUser({asTracker: false, username: state.config.username})
   )
 
-const addProof = (_, action) => {
+function* addProof(_, action) {
```

Comment:
before this was split into 2 functions that both handled the same event and used a switch to ensure only one ran, just make it one thing